### PR TITLE
chore(meta): add evm

### PR DESCRIPTION
**Pull request by canonbrother**
#### Which issue(s) does this PR fixes?:

Fixes #

### CODE REVIEW

File: meta/meta-runtime/src/lib.rs [hunk 1]
```diff
@@ -193,17 +228,87 @@ impl pallet_sudo::Config for Runtime {
...
+		gas.saturating_mul(WEIGHT_PER_GAS)
+	}
+	fn weight_to_gas(weight: Weight) -> u64 {
+		weight.wrapping_div(WEIGHT_PER_GAS)
+	}
+}
+
+parameter_types! {
+	// after looking the https://chainlist.org/, `988` is a good number since no other public network are using it
+	pub const ChainId: u64 = 988;
```
Comment (canonbrother) [inline meta/meta-runtime/src/lib.rs:243]: TBC

File: meta/meta-node/src/chain_spec.rs [hunk 2]
```diff
@@ -114,9 +146,47 @@ pub fn testnet_genesis(
...
+					fp_evm::GenesisAccount {
+						balance: U256::from_str("0xffffffffffffffffffffffffffffffff")
+							.expect("internal U256 is valid; qed"),
+						code: Default::default(),
+						nonce: Default::default(),
+						storage: Default::default(),
+					},
+				);
+				map.insert(
+					// H160 address of CI test runner account
```
Comment (surangap) [inline meta/meta-node/src/chain_spec.rs:174]: test runner 🤔
anw can change it when we use it in the CI

Comment (canonbrother) [reply meta/meta-node/src/chain_spec.rs:174]: yes.. use case: https://github.com/paritytech/frontier/blob/ef75c42e9eb65319fd09ae589e3d965633b2ef51/ts-tests/tests/config.ts#L1-L3

we can replace it via `subkey` tool as well
```sh
subkey generate
```

File: meta/meta-node/src/cli.rs [hunk 3]
```diff
@@ -31,4 +74,6 @@ pub enum Subcommand {
 
 	/// Revert the chain to a previous state.
 	Revert(sc_cli::RevertCmd),
+	/// Db meta columns information.
```
Comment (surangap) [inline meta/meta-node/src/cli.rs:77]:
```suggestion

	/// Db meta columns information.
```

File: meta/meta-runtime/src/lib.rs [hunk 4]
```diff
@@ -95,18 +123,22 @@ pub fn native_version() -> NativeVersion {
...
 parameter_types! {
 	pub const Version: RuntimeVersion = VERSION;
 	pub const BlockHashCount: BlockNumber = 2400;
-	/// We allow for 2 seconds of compute with a 6 second average block time.
 	pub BlockWeights: frame_system::limits::BlockWeights = frame_system::limits::BlockWeights
-		::with_sensible_defaults(2 * WEIGHT_PER_SECOND, NORMAL_DISPATCH_RATIO);
+		::with_sensible_defaults(MAXIMUM_BLOCK_WEIGHT, NORMAL_DISPATCH_RATIO);
 	pub BlockLength: frame_system::limits::BlockLength = frame_system::limits::BlockLength
 		::max_with_normal_ratio(5 * 1024 * 1024, NORMAL_DISPATCH_RATIO);
 	pub const SS58Prefix: u8 = 42;
```
Comment (surangap) [inline meta/meta-runtime/src/lib.rs:137]: change to `988` as you suggested? can change it if want to choose a different one entirely.

Comment (canonbrother) [reply meta/meta-runtime/src/lib.rs:137]: dropped all ss58prefix update... TBC

**Review by surangap — APPROVED** [review]
LGTM. Can deploy smart contract(with hardhat), interact with it. do token transfer from one account to another using metamask.

Need to check the polkadot.js failure and the submission via substrate-front-end-template. could be due to different reasons in the frontends.

**Pull request merged**